src/ui/widgets/alignwidget.cpp

```diff
@@ -121,6 +153,29 @@ AlignWidget::AlignWidget(QWidget* const parent)
     });
     buttonsLay->addWidget(bottomButton);
 
+    connect(mRelativeTo, QOverload<int>::of(&QComboBox::currentIndexChanged), this, [this, leftButton, rightButton, topButton, bottomButton]() {
+        if (mRelativeTo->currentIndex() == 2) {
```

Comment:
More "magic" numbers :)

Reply:
hehe, yes...

Reply:
"Last Selected Pivot" does not support all options?

Also, "Last Selected Pivot" does nothing(?).

Reply:
Si you mean in this piece of code or the feature itself.
The feature works and moves all pivots to match vertical/horizontally the last selected object pivot

Reply:
> The feature works 

Yes it does, my bad :smile: